### PR TITLE
EmberJS Cheat Sheet: Add EmberJS Route lifecycle methods

### PR DESCRIPTION
**Pull request by vikashvverma**
IA Page: https://duck.co/ia/view/emberjs_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=emberjs cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/emberjs_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=emberjs cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/emberjs_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/emberjs.json [hunk 1]
```diff
@@ -73,8 +73,29 @@
...
+                "key": "serialize: function("
+        }, {
+                "val": "Function that can be used to configure the controller",
+                "key": "setupController: function"
+        }, {
+                "val": "Object with properties",
+                "key": "actions"
+        }, {
+                "val": "Called whenever transition triggered on current route",
+                "key": "actions: { willTransition: function(transition) }"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/emberjs.json]: the `{` need to be escaped!

File: share/goodie/cheat_sheets/json/emberjs.json [hunk 2]
```diff
@@ -73,8 +73,29 @@
...
                 "key": "deactivate: function"
+        }, {
+                "val": "Provides data to be used by the controller and the view",
+                "key": "model: function"
+        }, {
+                "val": "hook to override default template rendered for this route",
+                "key": "renderTemplate: function"
+        }, {
+                "val": "Converts model into parameters for the url",
+                "key": "serialize: function("
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/emberjs.json]: We should normalize the use of `(` at the end. The other's don't put this, I think we should remove it elsewhere

**Pull request closed without merging**